po/en.po

```diff
@@ -256,10 +262,13 @@ msgid ""
 "때문에 특정 사용자가 리소스를 장기간 점유하는 문제를 해결하기에 어려움이 있습"
 "니다. 따라서 사용자 여러분들의 배려가 필요합니다."
 msgstr ""
+"Since the A100 GPU resources are limited, please only use it when necessary, such as when having a large deep learning model size or urgent training needs."
```

Comment:
```suggestion
"Since the A100 GPU resources are limited, please use them only when necessary, such as when having a large size deep learning model or needing to train some models urgently."
```